components/Header.js

```diff
@@ -1,28 +1,55 @@
-import React, { useCallback, useState } from "react";
+import React, { useCallback, useEffect, useState } from "react";
 import Link from "next/link";
 import { useRouter } from "next/router";
 import Router from 'next/router'
 import { magic } from '../magic'
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faHandSparkles } from '@fortawesome/free-solid-svg-icons'
+import { faMinusCircle, faPlusCircle } from "@fortawesome/free-solid-svg-icons";
 
 import Styles from "../styles/Components/Header.module.scss"
 
 const Header = ({ props }) => {
   const router = useRouter();
   const isActive = (pathname) => router.pathname === pathname;
-  const [showMobileLogout, setShowMobileLogout] = useState(false);
+  const [showMenu, setShowMenu] = useState(false);
+  const [fontSize, setFontSize] = useState();
+
+  const decText = () => {
+    let newSize = fontSize - 2;
+    document.getElementById('accessibleBody').style.fontSize = `${newSize}px`
```

Comment:
Wow, idk why it never occurred to me that you can use vanilla JS to select the DOM lmao